packages/@lwc/compiler/src/compiler/__tests__/modes.spec.ts

```diff
@@ -25,7 +25,7 @@ function getConfig(env = {}) {
     };
 }
 
-describe('environment replacement', function() {
+describe('environment replacement', function () {
```

Comment:
```suggestion
describe('environment replacement', () => {
```

Reply:
@ravijayaramappa I guess there are like a hundred of these and they don't really bother me. Do you think we should change them?

Reply:
I would suggest doing this in another PR.